tests/integ/test_local_mode.py

```diff
@@ -230,6 +251,53 @@ def test_tf_local_data_local_script():
         fcntl.lockf(local_mode_lock, fcntl.LOCK_UN)
 
 
+def test_local_mode_serving_from_s3_model(sagemaker_local_session, mxnet_model):
+    local_mode_lock_fd = open(LOCK_PATH, 'w')
+    local_mode_lock = local_mode_lock_fd.fileno()
+
+    model_data = mxnet_model.model_data
+    boto_session = sagemaker_local_session.boto_session
+    default_bucket = sagemaker_local_session.default_bucket()
+    uploaded_data = tar_and_upload_dir(boto_session, default_bucket,
```

Comment:
This is only necessary to create the model data, right? In the normal use cases, the user would either do both training and deploy locally, or create the MXNetModel pointing to local model data, or already have an S3 url with model data, right?

Reply:
yeah, this is because I wanted to make this faster by training using local mode instead of training on sagemaker like the chainer tests do for instance.